packages/babel-core/src/config/files/index-browser.js

```diff
@@ -12,16 +12,16 @@ export type IgnoreFile = {
   ignore: Array<string>,
 };
 
-export function findBabelrc(
+export type RelativeConfig = {
+  config: ConfigFile | null,
+  ignore: IgnoreFile | null,
+};
```

Comment:
Can this be reexported from `configuration.js`?

Reply:
Good call! I feel like there was a reason for duplicating these types like this originally, but I don't know what it was.